org.kde.kontact.json

```diff
@@ -0,0 +1,1233 @@
+                        }
+                    ]
+                },
+                {
+                    "name": "libassuan",
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-2.5.1.tar.bz2",
+                            "sha256": "47f96c37b4f2aac289f0bc1bacfa8bd8b4b209a488d3d15e2229cb6cc9b26449"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "kdiagram",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_INSTALL_LIBDIR=lib"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.kde.org/stable/kdiagram/2.6.1/kdiagram-2.6.1.tar.xz",
+                    "sha256": "bde71ca5ef70b3077696d56b24e0e1f69ae255c0bf7bf453e9596afdd25dcdb0"
+                }
+            ]
+        },
+        {
+            "name": "kcalcore",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_INSTALL_LIBDIR=lib"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/applications/18.08.1/src/kcalcore-18.08.1.tar.xz",
+                    "sha256": "563239f2bfc7c4727add80f621781917a1f280c545c4e19d8e98a68a6b14c94d"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "libical",
+                    "buildsystem": "cmake-ninja",
+                    "config-opts": [
+                        "-DCMAKE_INSTALL_LIBDIR=lib"
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://github.com/libical/libical/releases/download/v3.0.4/libical-3.0.4.tar.gz",
+                            "sha256": "72b216e10233c3f60cb06062facf41f3b0f70615e5a60b47f9853341a0d5d145"
+                        }
+                    ],
+                    "cleanup": [
+                        "/share/gtk-doc"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "boost",
+            "config-opts": [
+                "--with-libraries=graph,program_options,system,thread"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2",
+                    "sha256": "7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7"
+                },
+                {
+                    "type": "file",
+                    "path": "boost-configure",
```

Comment:
By using `buildsystem: simple` you could get ride of this file, see e.g.:
https://github.com/flathub/org.kde.krita/blob/master/org.kde.krita.yaml#L29